doc/getting-started/understanding-config-files.md

```diff
@@ -355,3 +355,19 @@ It is also possible to have more fine grained control over filtering of trace ou
 	  }
 	}
 ```
+
+#### UTxO HD
```

Comment:
Maybe we should call this something like:

- Ledger State HD 
- Ledger State on Disk

or something like that.